lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -143,10 +143,11 @@ sub list_ajax {
                 flavor  => $job->FLAVOR  // '',
                 arch    => $job->ARCH    // '',
                 build   => $job->BUILD   // '',
-                testtime => $job->t_finished . 'Z',
-                result   => $job->result,
-                group    => $job->group_id,
-                state    => $job->state
+                testtime      => $job->t_finished . 'Z',
+                result        => $job->result,
+                group         => $job->group_id,
+                comment_count => $job->comments->count,
```

Comment:
When writing this line I thought about performance, too. So I have already tested with production data and had no notable performance regression. I also tried nytprof but when displaying results my workstation crashed and then I forgot to continue. I can check that it again but making it server-side would be the proper solution of course. However, then I will finally have to refactor the code for that.